dev-lang/vala/vala-0.36.12.recipe

```diff
@@ -75,22 +97,18 @@ INSTALL()
 {
 	make install
 
-	# remove libtool library files
-	rm $libDir/lib*.la
+	rm -f $libDir/*.la
 
 	fixPkgconfig
 
-	packageEntries common \
-		$dataDir/aclocal/vapigen.m4 \
-		$dataDir/aclocal/vala.m4 \
+	# devel package
+	packageEntries devel \
+		$developDir \
+		$dataDir/aclocal \
 		$dataDir/vala/Makefile.vapigen
```

Comment:
```diff
+	prepareInstalledDevelLib libvala-$portVers
 	fixPkgconfig
 
 	# devel package
 	packageEntries devel \
 		$developDir \
 		$dataDir/aclocal \
-		$dataDir/vala/Makefile.vapigen
+		$dataDir/vala \
+		$dataDir/vala-$portVers
 }
```
I haven't checked it if we need to add the call to `prepareInstalledDevelLib`.
I also don't know if it's OK to put `$dataDir/vala-0.36` in the devel package.
BTW, I guess it would be nice to move to the devel package some commands (or maybe all) that are in the base package:
- [ ] vala
- [x] valac
- [x] vapicheck
- [x] vapigen